training/backends/kubernetes.js

```diff
@@ -138,7 +156,10 @@ module.exports = async function execTask(job, spec) {
                 metadata: {
                     labels: {
                         app: 'training-job',
-                        'edu.stanford.almond/job-id': job.id
+                        'edu.stanford.almond/job-id-task': jobIdTask
+                    },
+                    annotations: {
+                        'iam.amazonaws.com/role': 'arn:aws:iam::373474209952:role/almond-training-s3-admin'
```

Comment:
this should be pass through config

Reply:
We have a config key for extra metadata, we probably need one for extra annotations.

Reply:
done